github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.59.yaml

```diff
@@ -2279,7 +2279,45 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.26-sig-operator
+          value: k8s-1.26-sig-operator-upgrade
+        image: quay.io/kubevirtci/bootstrap:v20221222-8f66e7e
+        name: ""
+        resources:
+          requests:
+            memory: 29Gi
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - always_run: true
+    branches:
+    - release-0.59
+    cluster: prow-workloads
+    context: pull-kubevirt-e2e-k8s-1.26-sig-operator
```

Comment:
I think you need to update the context here to match the lanes on main

Reply:
Updated, both operator lanes